### PR TITLE
Explicitly clear out customers before testing

### PR DESCRIPTION
**Pull request by lanej0**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Fixes the e2e failure that we're seeing on the "On PR merge" workflow.  This explicitly clears out any existing customers before running the customer list tests.

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure CI passes

### Changelog entry

-   [ ] Automatically create a changelog entry from the details below.

<details>

#### Significance

-   [ ] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Comment 

</details>

### CODE REVIEW

File: plugins/woocommerce/tests/e2e-pw/tests/merchant/customer-list.spec.js [hunk 1]
```diff
@@ -78,6 +78,16 @@ baseTest.describe( 'Merchant > Customer List', () => {
 		},
 	} );
 
+	test.beforeAll( async ( { api } ) => {
+		let oldCustomers = {};
+		await api.get( 'customers' ).then( ( response ) => {
+			oldCustomers = response.data;
+		} );
+		await api.post( `customers/batch`, {
```
Comment (rrennick) [inline plugins/woocommerce/tests/e2e-pw/tests/merchant/customer-list.spec.js:86]: Will it create an api error if `oldCustomers` is empty?

**Review by rrennick — COMMENTED** [review]
@lanej0 This looks good except for that one question.

File: plugins/woocommerce/tests/e2e-pw/tests/merchant/customer-list.spec.js (hunk 1 above)
Comment (lanej0) [reply plugins/woocommerce/tests/e2e-pw/tests/merchant/customer-list.spec.js:86]: Nope, I tested it locally on my machine where there are no customers and no error is thrown.  So it works either way.

**Review by adimoldovan — APPROVED** [review]
Nice, I like the approach. 👍🏼

**Pull request merged**